### PR TITLE
fix: 270: retrieve the currently focused tab in any window

### PR DESCRIPTION
**Pull request by lasmil**
- closes https://github.com/nearform/github-snooze-chrome-extension/issues/270
- we need to get the currently focused tab if we have multiple windows open

### CODE REVIEW

File: src/api/chrome.js
Comment (simoneb) [inline src/api/chrome.js:243]: as I expected, this is one of the obvious use cases also shown in the docs https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab

**Review by simoneb — APPROVED** [review]

**Review by simoneb — CHANGES_REQUESTED** [review]
please don't mix unrelated things, especially when the unrelated thing turns out to be a lot more code than the issue that this pull request was supposed to fix. this is a general rule to apply all the time please. remove unrelated code, move that code to another PR.

**Comment by lasmil** [comment]
> please don't mix unrelated things, especially when the unrelated thing turns out to be a lot more code than the issue that this pull request was supposed to fix. this is a general rule to apply all the time please. remove unrelated code, move that code to another PR.

Alright, I can do that. But I think we need these changes as well to fix the hanging of running the storybook. Should I add these changes to the already open PR:
https://github.com/nearform/github-snooze-chrome-extension/pull/277

**Comment by simoneb** [comment]
We have another PR which fixes the build, this one stays broken until the other is merged. When the other is merged this can be rebased on master and the build will be green. Let's apply a little bit of discipline please.

**Comment by lasmil** [comment]
> We have another PR which fixes the build, this one stays broken until the other is merged. When the other is merged this can be rebased on master and the build will be green. Let's apply a little bit of discipline please.

I get that but the other PR does not fix the build without some changes from this PR. Unless I am not sure which PR are you speaking about. If it's this:
https://github.com/nearform/github-snooze-chrome-extension/pull/277
that also has the hanging storybook issue which is fixed in this PR (by having a separate script to run the storybook tests.

**Comment by simoneb** [comment]
I am simply talking about keeping the fix for the broken build and the fix for the active tabs in 2 different PRs

**Comment by lasmil** [comment]
> I am simply talking about keeping the fix for the broken build and the fix for the active tabs in 2 different PRs

My question was if should I apply the fix for the test hanging in Rafael's PR or not since it's not created by me and he needed to do a fork to create his PR. But I assume I can move the fix from this to his PR to get things going forward.

**Comment by simoneb** [comment]
Up to you, no preference about that

**Comment by lasmil** [comment]
> Up to you, no preference about that

We got to the same page in the end :) . I thought that there was some kind of limitation with this which is why I added the storybook test fixes to this PR in the beginning, 1 to see if the changes work and the build passes and 2 because by the time I got this working I was not able to mention it to Rafael.

Then the plan is this:
I will move my changes related to having the storybook tests pass to rafael's PR.

I will only keep the changes related to the bugfix in this PR.

**Review by simoneb — APPROVED** [review]
lgtm. we can merge this once build is green

**Pull request merged**
